lldb/include/lldb/Symbol/CompilerType.h

```diff
@@ -450,6 +450,9 @@ class CompilerType {
   /// Lookup a child member given a name. This function will match member names
   /// only and will descend into "clang_type" children in search for the first
   /// member in this class, or any base class that matches "name".
+  /// \param child_indexes returns an index path for the first result.
```

Comment:
Why do you say "FIRST result".  This finds the first occurrence of that name, but only returns one result.  So using "first" here is confusing.